docs/integration/opensearch-integration.md

```diff
@@ -40,6 +40,137 @@ OpenSearch runs within a Java Virtual Machine (JVM) and [monitoring JVM](https:/
 
 Finally, high disk reads and writes can indicate a poorly tuned system. Since accessing the disk is an expensive process in terms of time, a well-tuned system should reduce disk I/O wherever possible.
 
+
+## OpenSearch Default Alerts
```

Comment:
I assume those are the same as Elasticsearch for the moment.

Reply:
Yes, that's  right